hail_search/queries/mito.py

```diff
@@ -197,6 +198,8 @@ def _parse_variant_keys(self, variant_ids=None, **kwargs):
     def _prefilter_entries_table(self, ht, parsed_intervals=None, exclude_intervals=False, **kwargs):
         if exclude_intervals and parsed_intervals:
             ht = hl.filter_intervals(ht, parsed_intervals, keep=False)
+        elif len(parsed_intervals or []) >= MAX_LOAD_INTERVALS:
```

Comment:
do we know why this is faster?

[the docs](https://hail.is/docs/0.2/guides/genetics.html#using-hl-filter-intervals) say this is `suitable for a small list of intervals`, and we're doing the opposite.

Reply:
No idea, its very counterintuitive. I asked on zulip - https://hail.zulipchat.com/#narrow/stream/123010-Hail-Query-0.2E2-support/search/_intervals.20filter_intervals